Sources/BraintreeCoreSwift/BTHTTP.swift

```diff
@@ -306,8 +299,8 @@ import Security
             headers["Content-Type"] = "application/json; charset=utf-8"
         }
 
-        if tokenizationKey != "" {
-            headers["Client-Key"] = tokenizationKey
+        if case .tokenizationKey(let key) = authType {
+            headers["Client-Key"] = key
         }
```

Comment:
`if case let` is unusual syntax but I like that the enum forces you to check whether we're using an authorizationFingerprint or a tokenizationKey before you can even access it.